modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/schema/SqlSchemaManagerImpl.java

```diff
@@ -505,7 +540,7 @@ private static IgniteTableImpl createTable(
 
         // TODO IGNITE-19558: The table is not available at planning stage.
```

Comment:
the ticket should be closed after merge this one

Reply:
does it make sense to remove TODO in this patch?

Reply:
removed